src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/model/Policy.kt

```diff
@@ -68,6 +68,8 @@ data class Policy(
         builder.startObject()
         if (params.paramAsBoolean(WITH_TYPE, true)) builder.startObject(POLICY_TYPE)
         builder.field(POLICY_ID_FIELD, id)
+            // .field(SEQ_NO_FIELD, seqNo)
```

Comment:
Why are these fields being removed?

Reply:
I am also trying to figure it out... it turns out we just don't have these 2 field on our master branch. [code link](https://github.com/opendistro-for-elasticsearch/index-management/blob/master/src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/model/Policy.kt#L70-L76)

Reply:
Why are you trying to add them :P

Reply:
long time ago... I am also confused

Reply:
Oh yeah, I meant add :P